pyccel/codegen/printing/ccode.py

```diff
@@ -2251,7 +2255,9 @@ def _print_CustomDataType(self, expr):
         return "struct " + expr.name
 
     def _print_ClassDef(self, expr):
+        self._current_class = expr
```

Comment:
If you want to future-proof this, you may want to consider using a list and using the last element. This will then also handle classes in classes